wfe2/wfe.go

```diff
@@ -1134,6 +1142,14 @@ func (wfe *WebFrontEndImpl) Account(
 		return
 	}
 
+	if currAcct.Status == core.StatusDeactivated {
```

Comment:
Sorry @klebervirgilio I was wrong about needing this (See the [other review comments](https://github.com/letsencrypt/boulder/pull/4187/files#r280069127)). Could you undo this diff to `Account`?

Reply:
@cpu Sure! will undo it. The good thing is now we have a unit test for this path.

Reply:
@cpu is prob a good thing to keep it right?

Reply:
Yup! I think leaving the unit test makes sense and is a nice contribution :-)